pavlo976/BuildMinder-App
# Feature/employees table

### What's changed?
Employees table was created
### How to test it?
Open browser and try to sort, remove or add employee. Check if pagination correctly works.
### What's missing?
Employee filters have not yet been created.
